cognite/extractorutils/__init__.py

```diff
@@ -16,4 +16,4 @@
 Cognite extractor utils is a Python package that simplifies the development of new extractors.
 """
 
-__version__ = "1.0.3"
+__version__ = "1.2.0"
```

Comment:
Would it be possible to read this value from `pyproject.toml` to avoid having two divergent sources of truth here?

Reply:
I know, I really wish poetry somehow could populate this automatically, however the poetry folks have been [quite clear that they're never doing that](https://github.com/python-poetry/poetry/issues/144), and I have not found any way to work around this that I find sufficiently elegant.